lib/utils/avatar.vue

```diff
@@ -32,8 +32,14 @@
+        } else if (this.url && this.url.length && this.url !== 'https://lh3.googleusercontent.com/-XdUIqdMkCWA/AAAAAAAAAAI/AAAAAAAAAAA/4252rscbv5M/photo.jpg?sz=50') {
```

Comment:
what

Reply:
welcome to gooooogle
they're so swell
their default avi's that url
you think it's odd?
it's weird as hell